yt_dlp/cookies.py

```diff
@@ -146,7 +146,8 @@ def _extract_firefox_cookies(profile, container, logger):
             identities = json.load(containers).get('identities', [])
         container_id = next((context.get('userContextId') for context in identities if container in (
             context.get('name'),
-            try_call(lambda: re.fullmatch(r'userContext([^\.]+)\.label', context['l10nID']).group())
+            try_call(lambda: re.fullmatch(r'userContext([^\.]+)\.label', context['l10nID']).group()),
```

Comment:
also I know this is old code, but we never needed to escape the dot inside the character class
```suggestion
            try_call(lambda: re.fullmatch(r'userContext([^.]+)\.label', context['l10nID']).group()),
```